src/Reference/Commitments/SFA.DAS.Payments.Reference.Commitments.IntegrationTests/GivenApiAvailable/WhenReadingEvents.cs

```diff
@@ -637,6 +637,100 @@ public void ThenItShouldAddEventPriceHistoryWhereEffectiveFromIsBeforeEffectiveT
             Assert.AreEqual(1, history.Length);
         }
 
+        [Test]
+        public void ThenItShouldNotAddEventWhereEffectiveFromIsBeforeEffectiveTo()
+        {
+            var effectiveDate = DateTime.Today;
+
+            StubbedEventsApi.Events.Clear();
+            StubbedEventsApi.Events.Add(new ApprenticeshipEventView
+            {
+                Id = 3,
+                ApprenticeshipId = 2,
+                ProviderId = "369",
+                LearnerId = "258",
+                EmployerAccountId = "2",
+                TrainingType = TrainingTypes.Framework,
+                TrainingId = "854-965-621",
+                TrainingStartDate = new DateTime(2020, 9, 1),
+                TrainingEndDate = new DateTime(2021, 10, 1),
+                TrainingTotalCost = 99999,
+                PaymentStatus = Events.Api.Types.PaymentStatus.Withdrawn,
+                PaymentOrder = 99, //changed
```

Comment:
what is the purpose of this comment? is it needed?

Reply:
no - the bones of the test were copied from another and I didn't notice the comment. Will remove it.